samples/application-conf/application.properties

```diff
@@ -99,28 +99,28 @@ propertyFileExtension=properties
 #
 # boolean flag to configure the SearchPathDependencyResolver to evaluate if resolved dependencies impact
 #  the file flags isCICS, isSQL, isDLI, isMQ when creating the LogicalFile
-# 
-#  default:false 
+#
```

Comment:
@M-DLB Can you please have look what causes these different end of line chars?

Reply:
@M-DLB looks like that I have introduced these whitespaces to the application.properties in previous revisions.